### DISCUSSION TITLE
Center 306 on Chromebook linux Kaldi Fortis

### DISCUSSION THREAD

**Discussion by chinafields** [artisan-roaster-scope/artisan#612] [Q&A]
![1621456399363181035758078084314](https://user-images.githubusercontent.com/84476862/118880518-f4304300-b8bf-11eb-9de2-22b809ae0a80.jpg)
Hi I'm trying to set up my center 306 for my Kaldi Fortis with Artisan  and I keep getting Modbus Communication error or unable to open serial port. I have tried everything and keep having this issue. I was able to get it running on a MacBook, but I would like to use my pixel book to roast

**Comment by poundy** [comment]
As far as I know, there is no ChromeOS version of Artisan so your pixelbook would need to run a Linux variant that is supported. 

As for your meter, if you can get it to work on one platform, you should be able to get it to work on another. The Center 306 is a SERIAL device, not a MODBUS device, so it would seem you have misconfigured the pixelbook, given it’s reporting modbus errors.

One of the simplest ways you can get this working yourself is export the Artisan settings from the working configuration and from the not-working one, and compare them for differences in device configuration - this should tell you where you went wrong. I'd ordinarily suggest you just replace the not-working config with the working config but I am not sure whether you'll introduce issues because of your platform change here, at a minimum you could try that but make sure that you choose the appropriate serial port for your device on the pixelbook, as that will have changed from the mac.

**Reply by chinafields** [reply]
Yeah that was just one error when I tried to switch it. When I set it up for center 306 I get a serial port error of unable to open I have gone from 1-7 and every rate one by one yet I receive the same error.

**Reply by poundy** [reply]
the problem then is not Artisan, it's Linux. You need to troubleshoot why your device does not appear to the OS. Check out https://artisan-scope.org/docs/installation/ where CarefreeBuzzBuzz has some content on Linux permissions and has some of Rob's port naming details that might help you.

**Comment by CarefreeBuzzBuzz** [comment]
A few thoughts.  First the Center is  not a great device to start due  to  slow  sampling, but it does  work  ok.   Also  the proprietary cable is a pain to replace if lost or damaged. 

You could consider a Raspberry Pi set up 

https://artisan-roasterscope.blogspot.com/2019/02/running-artisan-on-raspberry-pi.html


----------------------------------------
Michael Herbert
***@***.***
Artisan RoasterScope Team

**Comment by fcebreros** [comment]
Hi,
what is your port and modbus setup?
Best Regards
Frank
